Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseOpMode.java

```diff
@@ -286,12 +301,23 @@ public double coneStackWidthMotorPower(double coneStackWidth) {
     }
 
     /**
-     * calculates the motor power for the robot drivetrain based on the pixel offset from the cone stack
+     * calculates the turning power for the robot drivetrain based on the pixel offset from the cone stack and the width of the cone stack
      * @param pixelOffset how far cone stack is horizontally from center of camera field of view in pixels
-     * @return motor power for robot drivetrain
+     * @param width how wide the cone stack is in the camera frame in pixels
+     * @return turning power for robot drivetrain
+     */
+    public double coneStackPixelsAndWidthToTurningPower(double pixelOffset, double width) {
+        return (Constants.CONE_STACK_CENTERING_PROPORTIONAL_KP * pixelOffset)/((1.0/250.0)*width);
+    }
+
+    /**
+     * calculates the strafing power for the robot drivetrain based on the pixel offset from the cone stack and the width of the cone stack
+     * @param pixelOffset how far cone stack is horizontally from center of camera field of view in pixels
+     * @param width how wide the cone stack is in the camera frame in pixels
+     * @return strafing power for robot drivetrain
      */
-    public double coneStackPixelsMotorPower(double pixelOffset) {
-        return Constants.CONE_STACK_CENTERING_KP * pixelOffset;
+    public double coneStackPixelsAndWidthToStrafingPower(double pixelOffset, double width) {
+        return (Constants.CONE_STACK_CENTERING_PROPORTIONAL_KP * pixelOffset)*((1.0/200.0)*width);
```

Comment:
Are these magic numbers? They are part of an equation and are only used once

Reply:
Yes. People need to understand where these numbers are coming from and what they mean. Using a variable with a clear name is really helpful. Also why is this different from the other one? Should they both be the same?

Reply:
resolved, added a constant